x-pack/plugins/lens/public/xy_visualization_plugin/types.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Position } from '@elastic/charts';
+import { ContextFunction, ArgumentType } from '../../../canvas/canvas_plugin_src/functions/types';
```

Comment:
The types have been merged into OSS world, can you use those instead?

Reply:
Let's do that as a separate PR, because I think the type signatures aren't aligned, and I'd rather do that after another master merge upstream.

Reply:
I'm currently working on that in the expression integration PR, so I will take care of it.